packages/wagmi/src/client.ts

```diff
@@ -194,6 +194,8 @@ export class Web3Modal extends Web3ModalScaffold {
 
     this.options = options
 
+    this.createMetadata(wagmiConfig)
```

Comment:
Why are we accessing and setting metadata from walletconnect connector, when it is available as an option? Plus same comment as in ethers package, this should be set from within scaffold's init method.